protocol/writing-resource/put-patch-auxiliary-resource.feature

```diff
@@ -0,0 +1,19 @@
+Feature: PUT or PATCH on auxiliary resources
+
+  Background: Set up container and parse link headers
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.createContainer()  
+    * def exampleTurtle = karate.readAsString('../fixtures/example.ttl')
+    * def rdfResource = testContainer.createResource('.ttl', exampleTurtle, 'text/turtle');
+
+  Scenario: PUT auxiliary resource to container
+    * def response = clients.alice.sendAuthorized('GET', container.url, null, null)
+    * def links = parseLinkHeaders(response.headers)
+    * def describedby = links.find(el => el.rel.toLowerCase() === 'describedby')
+    * def metaUrl = resolveUri(container.url, describedby.uri)
+    Given url metaUrl
+    And headers clients.alice.getAuthHeaders('PUT', metaUrl)
+    And header Content-Type = 'text/turtle'
+    And request "<./> a <#Something> ."
```

Comment:
Is `request` the correct term?

Reply:
I believe it is... That's what's documented and used elsewhere, but there may be nuances that are lost on me.

Reply:
Yes, `request` is how you refer to the `body` - not ideal but that's part of Karate. I don't think it needs `"` though -single quotes would be more consistent.